scripts/create_source_tarball.sh

```diff
@@ -22,7 +22,7 @@ function success() {
 }
 
 function warning() {
-    echo -e "\033[31mWARNING: $1\033[0m" >&2
+    echo -e "\033[33mWARNING: $1\033[0m" >&2
```

Comment:
This just makes warnings yellow rather than red, so they stand out from errors a bit better